### PR TITLE
Created ZoomInZoomOut animation file

### PR DESCRIPTION
**Pull request by siddoinghisjob**

### CODE REVIEW

**Comment by siddoinghisjob** [comment]
Kindly let me know if any changes are required. If not, then please accept my PR.

**Comment by siddoinghisjob** [comment]
@NiallEccles kindly accept my PR

**Comment by siddoinghisjob** [comment]
@NiallEccles Can you please review this and merge if its alright?

File: animations/ZoomInZoomOut.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+      .div{
+            display: flex;
+            justify-content: center;
+            align-items: center;
+            position: relative;
+            height: 100vh;
+            width:100wh;
+        }
```
Comment (NiallEccles) [inline animations/ZoomInZoomOut.css:8]: Can you move these into your CSS class?

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Thanks for making this PR.

Can you take a look a the changes I've requested? You also need to add your animation to the `animations.js` file

**Review by NiallEccles — CHANGES_REQUESTED** [review]
Can you add your animation to the `animations.js` file?

**Review by NiallEccles — APPROVED** [review]

**Pull request merged**